File: CSE1001/bob problem.py
n = float(input("Enter amount in hand: "))
c = float(input("Enter price of one chocolate: "))
m = float(input("Enter number of wrapper for free chocolate: "))

# compute number of chocolates bought
p = n//c
# compute number of free chocolates
f = p//m

print("Number of chocolates: ", int(p+f))

File: CSE1001/C/bob_problem.c
#include <stdio.h>

int main()
{
    int n, c, m;
    float noc;
    
    printf("Enter the amount in hand:");
    scanf("%d",&n);
    printf("Enter price of one chocolate:");
    scanf("%d",&c);
    printf("Enter the number of wrappers our dear little bob had:");
    scanf("%d",&m);
    noc= (n/c)+m;
    printf("%f",noc);
    
    return 0;
}
